src/IconButton/index.jsx

```diff
@@ -29,22 +31,31 @@ const IconButton = ({
       onClick={onClick}
       type="button"
     >
-      <FontAwesomeIcon
+      <IconComponent
```

Comment:
Will this be a breaking change for people already using this? It looks like no, but I just want to double check.

Reply:
Nope, no breaking changes here.